File: VB-Examples/09_PageSetup/SetGutterPosition.vb
Imports Spire.Doc

Namespace SetGutterPosition
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			' Create a new Document object
			Dim document As New Document()

			' Load the document from the specified file path
			document.LoadFromFile("..\..\..\..\..\..\Data\Sample.docx")

			' Access the first section of the document
			Dim section As Section = document.Sections(0)

			' Set the IsTopGutter property to True for the page setup of the section
			section.PageSetup.IsTopGutter = True

			' Set the Gutter width to 100.0F for the page setup of the section
			section.PageSetup.Gutter = 100.0F

			' Define the output file name for the modified document
			Dim output As String = "SetGutterPosition.docx"

			' Save the modified document to the specified output file path in Docx format
			document.SaveToFile(output, FileFormat.Docx)

			' Dispose of the Document object to release resources
			document.Dispose()

			'Launch the file
			WordDocViewer(output)
		End Sub

		Private Sub WordDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub

	End Class
End Namespace

File: CS-Examples/09_PageSetup/SetGutterPosition.cs
using System;
using System.Windows.Forms;
using Spire.Doc;

namespace SetGutterPosition
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            // Create a new Document object.
			Document document = new Document();

			// Load a Word document from a specified file path.
			document.LoadFromFile(@"..\..\..\..\..\..\Data\Sample.docx");

			// Get the first section of the document.
			Section section = document.Sections[0];

			// Set the top gutter option to true for the section's page setup.
			section.PageSetup.IsTopGutter = true;

			// Set the width of the gutter in points (100f).
			section.PageSetup.Gutter = 100f;

			// Specify the output file name for the modified document.
			string output = "SetGutterPosition.docx";

			// Save the modified document to the specified output file path in DOCX format.
			document.SaveToFile(output, FileFormat.Docx);
			
			// Dispose the existing document object.
			document.Dispose();

            //Launch the file
            WordDocViewer(output);
        }

        private void WordDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }

    }
}
